### PR TITLE
bugfix: framing error msg was not shown

### PR DESCRIPTION
**Pull request by scgbckbone**

### CODE REVIEW

File: ckcc/protocol.py [hunk 1]
```diff
@@ -259,52 +259,62 @@ class CCProtocolUnpacker:
...
     def okay(msg):
         # trivial response, w/ no content
         assert len(msg) == 4
         return None
 
     # low-level errors
+    @staticmethod
     def fram(msg):
-        raise CCFramingError("Framing Error", str(msg[4:], 'utf8'))
+        raise CCFramingError("Framing Error: " + str(msg[4:], 'utf8'))
```
Comment (scgbckbone) [inline ckcc/protocol.py:284]: this line is the actual fix --> `CCProtoError` wants just msg as an argument

Comment (scgbckbone) [reply ckcc/protocol.py:284]: otherwise just added `@staticmethod` decorator so it is clear that `msg` is not `self`

**Comment by doc-hex** [comment]
yup, fine

**Comment by scgbckbone** [comment]
merge (public)

**Pull request merged**